app/packages/core/src/plugins/SchemaIO/components/FileView.tsx

```diff
@@ -68,3 +82,16 @@ function fileToBase64(
     fileReader.onerror = (error) => resolve({ error });
   });
 }
+
+function humanReadableBytes(bytes: number): string {
```

Comment:
Can we move this to a util. This is duplicated in FileTable.tsx

Reply:
Yep